torch/csrc/autograd/variable.cpp

```diff
@@ -12,12 +12,14 @@ namespace torch { namespace autograd {
 Variable::Variable(
   std::unique_ptr<thpp::Tensor> data,
   bool requires_grad,
-  bool is_volatile)
+  bool is_volatile,
+  bool sparse_grad)
     : data(std::move(data))
     , creator(nullptr)
     , grad(nullptr)
     , version_counter(new VariableVersion())
     , output_nr(0)
+    , sparse_grad(0)
```

Comment:
Good catch - looks like the two mistakes you pointed out cancelled each other :)